dask/tests/test_threaded.py

```diff
@@ -105,58 +104,35 @@ def test_f():
     assert L == [1] * 20
 
 
-code = """
-import sys
-import time
-from dask.threaded import get
-
-def signal_started():
-    sys.stdout.write('started\\n')
-    sys.stdout.flush()
+def test_interrupt():
+    if not PY3:
```

Comment:
Also might be better for this to be an explicit check for `PY2` as opposed to `not PY3`.